features/step_definitions/basic_steps.rb

```diff
@@ -0,0 +1,22 @@
+Given("I am on the {string} Page") do |page_name|
+  visit page_path(page_name)
+end
+
+Then("I fill in {string} with {string}") do |element, content|
+  fill_in(element, :with => content)
+end
+
+Then("I click on {string}") do |button_name|
+  click_link_or_button(button_name)
+end
+
+Then("I should see {string}") do |message|
+  page.should have_content message
```

Comment:
Please use expect to make assertions, as we’ve done through the entire course

Reply:
Will fix that. 
We forgot and looked in the capybara documentation.